dropdownDB.php

```diff
@@ -1,14 +1,28 @@
 <?php
 
-
+/**
+ *The function takes 2 parameters (database PDO and string) runs an SQL query
```

Comment:
Not sure, but think in doc blocks this should be more descriptive, e.g. 'gets article titles from the Db'